python/dfu.py

```diff
@@ -116,10 +116,11 @@ def reset(self):
   def program_bootstub(self, code_bootstub):
     self._handle.clear_status()
 
-    # erase all sectors
-    for i in range(len(self._mcu_type.config.sector_sizes)):
+    # erase bootstub + app sectors
+    for i in (0, 1):
```

Comment:
this is going to break again once we get a binary that's larger than these sectors, why not revert to the generic one?

Reply:
maybe we never had that calc in dfu: https://github.com/commaai/panda/blob/18230831f311a553ef64e55c1f418cb155eb3d7d/python/__init__.py#L432